### ISSUE TITLE
GitHub linguist fails to recognize Processing

### ISSUE THREAD

**Issue by jeremydouglass** [jeremydouglass/rosetta_examples_p5#54]
The language composition report is missing Processing on the repo header and in notification badges etc. -- this inhibits discovery by the Processing community.

Issue submitted here:

https://github.com/github/linguist/issues/4829

**Comment by jeremydouglass** [comment]
I was able to resolve the issue -- the path /examples/ was matching `linguist-documentation` so I was able to exclude it with:

```
*.pde -linguist-documentation
```

...and then simplified it to this, which also works and is a more general fix for the problem:

```
examples/* -linguist-documentation
```

**Comment by jeremydouglass** [comment]
Closed by c775c33
